frameworklauncher/conf/frameworklauncher.yml

```diff
@@ -25,6 +25,7 @@ amRmResyncFrequency: 6
 amRequestPullIntervalSec: 30
 amStatusPushIntervalSec: 30
 amFrameworkInfoPublishIntervalSec: 30
+amGangAllocationTimeoutSec: 500
```

Comment:
>500 [](start = 28, length = 3)

500， different 1100 in pai-management, which one is final number ?

Reply:
1100 is the final number on k8s

---
In reply to: [199710525](https://github.com/Microsoft/pai/pull/798#discussion_r199710525) [](ancestors = 199710525)